scrapy_zyte_api/utils.py

```diff
@@ -1,6 +1,10 @@
 import scrapy
 from packaging.version import Version
 
+from .__version__ import __version__
+
+USER_AGENT = f"scrapy-zyte-api/{__version__}"
```

Comment:
I think it might be a bit confusing, because that't not actually the user agent which is used, as the real user agent includes python-zyte-api version.

Reply:
@kmike what about rename this to SCRAPY_ZYTE_API_USER_AGENT? A little long though...

Reply:
It seems you can store the complete user agent as a constant, why do you need a constant with a part of the user agent?

Reply:
you are right @kmike , that's better, thanks - fixed that. One question left here https://github.com/scrapy-plugins/scrapy-zyte-api/pull/130#issuecomment-1737095570

Reply:
> One question left here https://github.com/scrapy-plugins/scrapy-zyte-api/pull/130#issuecomment-1737095570

I think that's better to bump python-zyte-api min version requirement. We want users to use the latest client library anyways, because it may contain fixes for the client library behavior.